site/components/map/StopsMap.tsx

```diff
@@ -99,9 +100,18 @@ const Map = ({
         if (features && Object.values(features).length > 0) {
             const polygon = Object.values(features)[0].geometry.coordinates[0];
             const loadOptions = async () => {
+                const vehicleMode = state.inputs.vehicleMode as Modes | VehicleMode;
                 const stopsData = await fetchStops({
                     adminAreaCodes: state.sessionWithOrg?.adminAreaCodes ?? ["undefined"],
                     polygon,
+                    ...(vehicleMode === VehicleMode.bus ? { busStopType: "MKD" } : {}),
+                    ...(vehicleMode === VehicleMode.bus
+                        ? { stopTypes: ["BCT"] }
+                        : vehicleMode === VehicleMode.tram || vehicleMode === Modes.metro
+                        ? { stopTypes: ["MET", "PLT"] }
+                        : vehicleMode === Modes.ferry || vehicleMode === VehicleMode.ferryService
+                        ? { stopTypes: ["FER", "FBT"] }
+                        : { stopTypes: ["undefined"] }),
```

Comment:
No data is returned for trains because stopTypes is set to undefined string

Reply:
as mentioned above